society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -938,10 +938,20 @@
       <choose>
         <!-- Support custom citations via the annote variable. For discussion, see https://forums.zotero.org/discussion/comment/317370 -->
         <if variable="annote">
-          <group delimiter=" ">
-            <text variable="annote"/>
-            <text macro="point-locators-subsequent"/>
-          </group>
+          <choose>
+            <if locator="sub-verbo">
```

Comment:
This looks good in principle, but from your description I don't understand why this is limited to "sub-verbo"? Shouldn't it be for all locators?

Reply:
Fair point. [SBL style requires no comma](https://sblhs2.com/2017/04/20/citing-reference-works-7-greek-language-tools/) with section-based citations (thus, e.g., "BDF §123"). Sub verbo is probably the most common contrary case where the comma _should_ appear.

But it's probably simpler test for whether the locator is a section and then omit the comma, including it everywhere else. There might be some other places where it should be omitted, but it looks at least like [page-based citations with custom abbreviations should have the comma too](https://sblhs2.com/2017/03/30/citing-reference-works-2-lexica/) (e.g., "BDB, 562a."). So, having the comma is probably the better catch-all. I'll update the request presently.

Reply:
Though, there's a wrinkle in this approach: Out of the sources that SBL style specifies abbreviation-based citations for, some also use page number as a locator. But among these, some include a comma and space delimiter (as in the example above). Others include only a space (e.g., [PL](https://sblhs2.com/2017/01/31/mignes-patrologia-latina/) and [PG](https://sblhs2.com/2017/05/02/pg-citations/)).

The only way I'm thinking of for the software to distinguish when to use one or the other delimiter is by testing the value of the `annote` variable. But am I right that that's [neither currently possible nor likely to happen in the foreseeable future](https://github.com/citation-style-language/csl-evolution/issues/26)?

If so, would you see any other options for disambiguating between when to give the comma+space delimeter and when the space only version?